src/apps/backend/modules/share-task-request/rest-api/share-task-request-router.ts

```diff
@@ -0,0 +1,15 @@
+import { accessAuthMiddleware } from '../../access-token';
+import { ApplicationRouter } from '../../application';
+import { ShareTaskRequestController } from './share-task-request-controller';
+
+export default class ShareTaskRequestRouter extends ApplicationRouter {
+  configure(): void {
+    const { router } = this;
+    const controller = new ShareTaskRequestController();
+
+    router.use(accessAuthMiddleware);
+
+    router.post('/tasks/:taskId/share-task-requests', controller.createSharedTask);
```

Comment:
Add `/tasks/:taskId/share-task-requests` to  `src/apps/backend/modules/share-task-request/rest-api/share-task-request-server.ts` file as it is a base URI.